commons/src/main/java/cloud/orbit/concurrent/Task.java

```diff
@@ -793,7 +790,7 @@ public void run()
      */
     public static <F extends CompletableFuture<?>> Task<Object> anyOf(Stream<F> cfs)
     {
-        return from(CompletableFuture.anyOf((CompletableFuture[]) cfs.toArray(size -> new CompletableFuture[size])));
+        return from(CompletableFuture.anyOf((CompletableFuture[]) cfs.toArray(CompletableFuture[]::new)));
```

Comment:
Byte code is identical, lambda method reference looks nicer though :)

Reply:
![screenshot from 2018-01-17 13-36-07](https://user-images.githubusercontent.com/323497/35434786-a945705e-0288-11e8-8f91-c35c7ba75323.png)